src/Condition/QueryEvaluator.php

```diff
@@ -154,7 +154,29 @@ protected function _parseCondition($field, $operator, $value)
             $field = $this->_config['_prefix'] . '.' . $field;
         }
 
```

Comment:
there is no sense in modifying this method in QueryEvaluator class. why?

Reply:
because _parseCondition method populates this output and if i do create a separate method in QueryEvaluatorTwofold class that will override this, it will not make sense too. it will be redundant in the QueryEvaluatorTwofold class since the update is just a couple lines of codes not like parseCondition method that i have in this class, it's totally different from parseCondition method in QueryEvaluator class.

Reply:
you're right. that's the reason, why you should create this method in QueryEvaluatorTwofold class and make a `parent::` call there.

Reply:
i've moved the parsing of text in QueryEvaluatorTwofold class and reverted QueryEvaluator class.